lib/libtrakt/exceptions.py

```diff
@@ -1,8 +1,43 @@
-class traktException(Exception):
-    pass
+from requests.exceptions import (ConnectionError, Timeout, TooManyRedirects)
 
-class traktAuthException(traktException):
-    pass
 
-class traktServerBusy(traktException):
-    pass
+class TraktException(Exception):
+    """A Generic Trakt Exception"""
+
+
+class TraktAuthException(TraktException):
+    """A Generic Trakt Authentication Exception"""
+
+
+class TraktServerBusy(TraktException):
+    """A Generic Trakt Server Busy Exception"""
```

Comment:
Not all the doc strings need to be "A Generic...".  For example this is a specific exception for the Trakt server being busy.